fish_completions.go

```diff
@@ -45,7 +45,7 @@ function __%[1]s_perform_completion
     __%[1]s_debug "Starting __%[1]s_perform_completion"
 
     # Extract all args except the last one
-    set -l args (commandline -opc)
+    set -l args (commandline -opc | string escape)
```

Comment:
cc @marckhouzam 

FWIW the blame for this mistake is on me, I made the [suggestion to use it](https://github.com/spf13/cobra/pull/1249#discussion_r563337287)